table_test.py

```diff
@@ -308,6 +331,49 @@ def test_insert_with_only_primary_key_no_kwargs(self, connection_function):
 
         table_count = mro.table1()
 
+    def test_disable_insert_thread_safe(self, connection_function_for_threadsafe_test):
```

Comment:
Have confirmed that the test fails when we don't use thread-local storage and passes when we do